printer_mods/watt-this/in-situ-lubrication/README.md

```diff
@@ -0,0 +1,232 @@
+# In Situ Linear Carriage Block Lubrication
+
+<table style="border-style:none">
+   <tbody>
+      <tr>
+         <td rowspan="6">
+            <img src="./assets/in_situ_regreasing.webp">
+         </td>
+         <td colspan="4">
+            With the advent of the faster and faster printers, the Voron printers principle among these, the manufacturers recommended greasing intervals becomes increasingly important.<br>
+            <br>
+            For MGN09 and MGN12 rails, <a href="https://hiwin.us/wp-content/uploads/lubricating_instructions.pdf">HIWIN</a> recommends greasing the rail after approximately 130 and 160 km (~80 and 100 miles) of movement respectively.<br>
+            For MGN07 the distance is 100km (62 miles).<br>
+            <br>
+            If you used oil instead of grease to lubricate the rails, the distance is reduced by 50% (and for low viscosity grease, greasing intervals are reduced to about 75%).
+         </td>
+      </tr>
+      <tr>
+         <td>
+            <b>Rail</b>
+         </td>
+         <td>
+            <b>Grease</b>
+         </td>
+         <td>
+            <b>Low Viscosity Grease</b>
+         </td>
+         <td>
+            <b>Oil</b>
+         </td>
+      </tr>
+      <tr>
+         <td>
+            MGN07
+         </td>
+         <td>
+            100km (62 miles)
+         </td>
+         <td>
+            75km (47 miles)
+         </td>
+         <td>
+            50km (31 miles)
+         </td>
+      </tr>
+      <tr>
+         <td>
+            MGN09
+         </td>
+         <td>
+            130km (81 miles)
+         </td>
+         <td>
+            98km (61 miles)
+         </td>
+         <td>
+            75km (47 miles)
+         </td>
+      </tr>
+      <tr>
+         <td>
+            MGN12
+         </td>
+         <td>
+            160km (100 miles)
+         </td>
+         <td>
+            120km (75 miles)
+         </td>
+         <td>
+            85km (53 miles)
+         </td>
+      </tr>
+      <tr>
+         <td colspan="4">
+            (with load ratio P/C of 0)
+         </td>
+      </tr>
+   </tbody>
+</table>
```

Comment:
I was trying to make the readme more informative by adding image in the introduction without breaking the flow of the information. It was one of the comments I got on Discord, that it was difficult to understand how the mod worked. I couldn't get the layout to work standard markdown tables. Reading through this, did the instruction otherwise from through clear enough to communicate how the mod work and how to implement it?